Discord bot

DJettt opened the pull request:
+Discord echo bot активен, но он не распознает команду /help, так как на данном этапе выполнения работы это нам не нужно.

DJettt committed:
Добавлено:
+Discord echo bot активен, но он не распознает команду /help, так как на данном этапе выполнения работы это нам не нужно.

ASHGOLDOFFICIAL commented on src/main/java/ru/urfu/Main.java:
```diff
@@ -1,25 +1,59 @@
+        }).start();
+    }
+    private static void startDsBot(){
+        EchoBotDiscord bot = new EchoBotDiscord();
```
Создавай бота после проверки токена.

DJettt committed:
Исправление ошибок
Добавлена оптимизация при создании Discord бота.

ASHGOLDOFFICIAL commented:
Всё окей.

DJettt committed:
Merge remote-tracking branch 'origin/echo-bot' into discord-echo-bot

# Conflicts:
#	src/main/java/ru/urfu/Main.java

DJettt committed:
Вынесение логики из дискорд бота.
Теперь на обоих ботов существует одна общая отдельно вынесенная логика генерации сообщений.

ASHGOLDOFFICIAL commented:
Стало ещё лучше.
